manifests/state-driver/0500_daemonset.yaml

```diff
@@ -572,6 +572,12 @@ spec:
           - name: host-os-release
             mountPath: /host-etc/os-release
             readOnly: true
+        {{- if eq .Driver.Spec.DriverType "vgpu-host-manager" }}
```

Comment:
Don't we need the same if conditional block for the `volumes` as well?

Reply:
No, we always add these volumeMounts in the main container, so no need for a conditional in the `volumes` section. Only for `vgpu-host-manager` do we add this volume mount to the driver-toolkit container.

Reply:
Understood. In that case, why make it a conditional at all? There shouldn't be any harm having the driver toolkit mount this volume at all times just like the main container, yes?

Reply:
I don't think there would be any side effects, but would need to verify.

Reply:
I prefer to limit the "if conditional blocks" as much as possible. Moreover, this would also be consistent with how we mount volumes in the main container

Reply:
Removed the conditional.

Reply:
Thanks!